daemon/daemon_unix.go

```diff
@@ -38,6 +38,38 @@ const (
 	platformSupported = true
 )
 
+func getBlkioReadIOpsDevice(config *runconfig.HostConfig) ([]string, error) {
```

Comment:
This and `getBlkioWriteIOpsDevice` look exactly the same. Can we instead pass a value instead of runconfig.HostConfig?